src/lib/create/from-string.js

```diff
@@ -105,13 +213,19 @@ export function configFromString(config) {
     configFromISO(config);
     if (config._isValid === false) {
         delete config._isValid;
-        hooks.createFromInputFallback(config);
```

Comment:
this code should be linear but is instead nested. I'd return in case isValid becomes true, so more parsers can be added without increasing indentation.